### ISSUE TITLE
Mute paths from the event screen

### ISSUE THREAD

**Issue by theevilbit** [redcanaryco/mac-monitor#21]
**At a high level -- can you summarize your request?**
It would be good to be able to mute paths from the event screen, not just filtering.

**What is the current alternative solution?**
Go into settings and manually add muted path.

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
NA

**Anything else?**
NA

**Comment by Brandon7CC** [comment]
Hey @theevilbit! Thank you for submitting this -- adding additional ways to mute paths is something I'm super interested in while balancing the UI/UX! 

Right now users have the option to mute paths by right clicking on an event and clicking "Mute path". Would you suggest any other improvements? 
<img width="447" alt="Screenshot 2023-04-28 at 11 01 07 AM" src="https://user-images.githubusercontent.com/22716543/235220931-918064c8-61ef-4ca2-8638-56fef939fee5.png">

**Comment by theevilbit** [comment]
Hi,

Ah, yes, I meant mute the target path similarly to the other feature request, where I would filter the target path.

Thanks,
Csaba

**Comment by Brandon7CC** [comment]
Done and implemented in `v1.0.4` thank you for the excellent suggestion!
